docs/src/pages/[platform]/getting-started/usage/usage.angular.mdx

```diff
@@ -0,0 +1,7 @@
+import { Alert, Text } from '@aws-amplify/ui-react';
+
```

Comment:
The messaging on these Angular, Vue, and Flutter usage pages could use some review (they are all sort of the same). Do we even want to mention the legacy components or just drive people  to the Authenticator docs?

Reply:
Probably just drive people people to the Authenticator docs

Reply:
Updated the not-React platforms to point to the Authenticator docs in latest.